src/style.css

```diff
@@ -1 +1,34 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.quality {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url('images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars--0 > :nth-child(0),
```

Comment:
The selector ':nth-child(0)' is not valid as there is no 0th child in CSS. Consider using ':first-child' to target the first star in the '.stars--0' class.